code/dce-downloader/node/README.MD

```diff
@@ -0,0 +1,233 @@
+# DCE Downloader Tools in Node.js
+
+Two Node.js scripts files are provided to get data from the Bazaarvoice Displayable Content Export service. Please refer to [documentation](https://developer.bazaarvoice.com/displayable-content-export) for details.
+
+## Prerequisite
+1. You should have Node.js 5.0 or later installed.
+2. You should install the package dependencies
+3. You must have a config.json file (see [Common Prerequisites](../README.MD#Common-Prerequisites))
+    
+## Development
+
+To get some background on working with Python, check out [this blog post](http://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/) and [this virtualenv doc](http://docs.python-guide.org/en/latest/dev/virtualenvs/#virtualenvwrapper).
+
+### MacOS Environment
+
+In your terminal, execute the following:
+
+```bash
+# Install node
+brew update
+brew install node
+```
+
+Install DCE package dependencies:
+
+```bash
+cd code/dce-downloader/node
+npm install --save
```

Comment:
you don't need the `--save`. It's only used when installing a new dependency and including it in the `dependencies` section of the `package.json`. Running `npm install` will install all the dependencies already listed under the `dependencies` section